api/PushSubscription.json

```diff
@@ -314,7 +314,7 @@
               "version_added": "17"
             },
             "firefox": {
-              "version_added": "46"
+              "version_added": "44"
             },
             "firefox_android": {
               "version_added": "48"
```

Comment:
Should this also be updated? It didn't match before, but should it?

Reply:
It might need to; the original data came from the wiki migration.  I'm not currently able to test older Firefox Android versions to confirm or deny this, though.

Reply:
Hmm, there's other data for https://w3c.github.io/push-api/ that says Firefox 48, even with some odd notes, so this is probably correct.